forui/lib/src/widgets/text_field/text_field.dart

```diff
@@ -123,6 +123,9 @@ final class FTextField extends StatelessWidget with FFormFieldProperties<String>
   /// See [TextField.statesController] for more information.
   final WidgetStatesController? statesController;
 
+  /// {@macro flutter.widgets.editableText.obscuringCharacter}
```

Comment:
We generally don't use macros that reference external packages/Flutter as it doesn't render properly in AS/IntelliJ's preview window.

I think we should copy-paste the description instead.